src/tests/VisualStudioProjectLoaderTests.cs

```diff
@@ -296,6 +296,35 @@ public void FromSolutinoWithProjectUsingPackageReference()
                 Assert.AreEqual(1, project.GetTestPackage("Release").SubPackages.Count, "Release should have 1 assemblies");
                 Assert.AreEqual(1, project.GetTestPackage("Debug").SubPackages.Count, "Debug should have 1 assembly");
             }
+        }
+
+        [Test]
+        public void FromProjectWithParametrizedPaths()
+        {
+            using (var file = new TestResource("TestDynamicPaths.csproj", NormalizePath(@"csharp-sample\csharp-sample.csproj")))
+            {
+                IProject project = _loader.LoadFrom(file.Path);
+                Assert.AreEqual(3, project.ConfigNames.Count);
+
+                var debugPackage = project.GetTestPackage("Debug");
+                Assert.AreEqual(1, debugPackage.SubPackages.Count, "Debug should have 1 assembly");
+
+                Assert.That(!debugPackage.SubPackages[0].FullName.Contains(@"$(Configuration)"),
+                    "Assembly path contains '$(Configuration)' which should be replaced with config name.");

```

Comment:
This was initially confusing because my eye skipped the `!`. You'll get a better error message and it will be clearer if you test with `...Not.Contains...`.

That is...
```C#
Assert.That(debugPackage.SubPackages[0].FullName, Does.Not.Contain(@"$(Configuration)");
```

Reply:
I've changed it.